optin/reporting/test/controllers/versionController.spec.js

```diff
@@ -0,0 +1,48 @@
+const Config = require('../../config');
+const VersionController = require('../../controllers/versionController');
+const TestUtils = require('../TestUtils');
+const { expect } = require('chai');
+const { stub, restore } = require('sinon');
+const path = require('path');
+
+describe('version controller', () => {
+	afterEach(restore);
+
+	const runBasicVersionTests = async ({ configPath, expectedResult }) => {
+		// Arrange:
+		const req = {};
+		const res = TestUtils.mockResponse();
+
+		stub(Config, 'getDataStoragePath').returns(path.join(__dirname, configPath));
+
+		// Act:
+		await VersionController.getVersion(req, res);
+
+		// Assert:
+		const result = res.json.getCall(0).firstArg;
+
+		expect(result).to.be.eql(expectedResult);
+	};
+
+	describe('getVersion', () => {
+		it('return version id and last update date time', async () => {
+			await runBasicVersionTests({
+				configPath: '../mock',
+				expectedResult: {
+					versionId: '_ykRdiKlyoCZJAi4FPnptUcNL5FeNW9G',
+					lastUpdated: 'Tue, 10 May 2022 13:59:22 GMT'
+				}
+			});
+		});
+
+		it('return version id and last update with empty value if file not found', async () => {
```

Comment:
is this expected or why is this not an error / exception?

Reply:
it's expected, that `current.version` is not found, it will return an empty value.